.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
If you are wondering what is this. These are vsCode settings, that you applied or were aplied automatically.
`.vscode` folder should be in `.gitignore`